app/controllers/admin/deduplicate_patients_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::DeduplicatePatientsController < AdminController
+  skip_before_action :verify_authenticity_token
+  DUPLICATE_LIMIT = 250
+
+  def show
+    authorize { current_admin.power_user? }
+
+    duplicate_patient_ids = PatientDeduplication::Strategies.identifier_match(limit: DUPLICATE_LIMIT)
+    @duplicate_count = duplicate_patient_ids.count
+    @patients = Patient.where(id: duplicate_patient_ids.sample)
+  end
+
+  def merge
+    authorize { current_admin.power_user? }
+
+    duplicate_patients = Patient.where(id: params[:duplicate_patients])
+    PatientDeduplication::Deduplicator.new(duplicate_patients, user: current_admin).merge
```

Comment:
It does return the final patient, and we tried including the patient name in the flash. But felt it might actually be a distraction rather than useful since the names might not match, and we just pick the latest of them. Wdyt?

Reply:
My instinct was, it would be nice to see which patient record "survives". I'll defer that to Claudio and Daniel, though. I don't have a strong opinion either way.

Reply:
@dburka @claudiovallejo thoughts on what the flash should show when a set of patients is merged?

Reply:
Including the name sounds like a good idea (I know it might not be the one you wanted to keep @ssrihari ). That's all I would include tho, since you have no control to edit the data after the merge.

Reply:
Fixed in 5f2da8a76ac5fa674d3dade63c436be8ade23ffc.